src/profile/components/modals/confirmationModal/ConfirmationModal.tsx

```diff
@@ -1,21 +1,21 @@
 import React from 'react';
 import ReactModal from 'react-modal';
 import { useTranslation } from 'react-i18next';
-import { IconCross } from 'hds-react';
+import { IconCross, IconAlertCircle } from 'hds-react';
 
 import styles from './ConfirmationModal.module.css';
-import { ServiceConnectionsQuery } from '../../../../graphql/generatedTypes';
+import { ServiceConnectionsRoot } from '../../../../graphql/typings';
 import getServices from '../../../helpers/getServices';
 import Button from '../../../../common/button/Button';
 
-type Props = {
+export type Props = {
   isOpen: boolean;
   onClose: () => void;
   onConfirm: () => void;
   modalTitle?: string;
   modalText?: string;
   actionButtonText: string;
-  services?: ServiceConnectionsQuery;
+  services?: ServiceConnectionsRoot;
 };
 
 function ConfirmationModal({
```

Comment:
We might be able to apply HDS Dialog here: https://hds.hel.fi/components/dialog ?

Reply:
Hope so! But it does not exist in current HDS version used here. Should be replaced in the HDS update ticket: https://helsinkisolutionoffice.atlassian.net/browse/HP-727